rust/telemetry/src/feature_flags.rs

```diff
@@ -118,19 +125,19 @@ async fn decide(maybe_legacy_id: String, api_key: String) -> Result<FeatureFlags
         .context("Failed to send POST request")?;
 
     let status = response.status();
+    let body = response.text().await.unwrap_or_default();
```

Comment:
Will this behave as expected for empty or errored response bodies?

Reply:
If this fails, we will hit a deserialisation error which - same as all other errors during feature flag evaluation - are logged on debug and then discarded. Shouldn't cause any issues.